administration/crd-extending-api-with-crds.adoc

```diff
@@ -0,0 +1,20 @@
+[id='crd-extending-api-with-crds']
+= Extending the Kubernetes API with Custom Resource Definitions
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:context: crd-extending-api-with-crds
+
+toc::[]
+
+include::modules/crd-custom-resource-definitions.adoc[leveloffset=+1]
+include::modules/crd-creating-crds.adoc[leveloffset=+1]
+include::modules/crd-creating-aggregated-cluster-roles.adoc[leveloffset=+1]
+include::modules/crd-creating-custom-resources-from-web-console.adoc[leveloffset=+1]
```

Comment:
I'm not delighted about including alternate ways to complete a task in a single flow. If you need to include both, you should add a short subsection in the assembly that explains that you can do either, and nest the modules under it.